docs/reference-guides/rancher-webhook.md

```diff
@@ -38,3 +58,34 @@ When using a private GKE cluster, errors may occur that prevent the Kubernetes A
 Internal error occurred: failed calling webhook "rancher.cattle.io.namespaces.create-non-kubesystem": failed to call webhook: Post "https://rancher-webhook.cattle-system.svc:443/v1/webhook/validation/namespaces?timeout=10s": context deadline exceeded
 ```
 This issue occurs because firewall rules restrict communication between the API server and the private cluster. To resolve this communication problem, users must add firewall rules to allow the GKE control plane to communicate with the Rancher-Webhook on port 9443. Please refer to the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) for detailed information and steps on updating the firewall rules.
+
+### Application Fails to Deploy due to rancher-webhook blocking access
+
+The webhook provides extra validations on [namespaces](https://github.com/rancher/webhook/blob/release/v0.4/docs.md#psa-label-validation). One of these validations ensures that users can only update PSA relevant labels if they have the proper permissions (`updatepsa` for `projects` in `management.cattle.io`). This can result in specific operators (such as tigera or trident) which attempt to deploy namespaces with PSA labels failing to deploy these resources. There are several ways that this issue can be resolved:
+
+- The application (e.g. tigera/trident) should be configured to create a namespace with no PSA permissions/labels. If users which to apply a PSA to these namespaces, they can add them to a project with the desired PSA after configuration. See the [docs on PSS and PSA resources](../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards) for more information on how to perform this operation.
+  - This is the preferred option, though not all applications can be configured in this fashion.
+- Users can manually grant the operator permissions to manage PSAs for namespaces.
+  - This option will introduce new security risks, since this operator will now be able to set the PSA for the namespaces it has access to. This could allow the operator to deploy a privileged pod (or effect cluster takeover through other means).
```

Comment:
```suggestion
  - This option will introduce security risks, since the operator will now be able to set the PSA for the namespaces it has access to. This could allow the operator to deploy a privileged pod, or effect cluster takeover through other means.
```